Record/Model/MusicAPI.swift

```diff
@@ -9,32 +9,28 @@
 import SwiftUI
 
 // MARK: - 음악 API 불러오기
+@MainActor
 class MusicAPI: ObservableObject {
+    
+    // 네트워크 접근 싱글톤 객체
+    static let searchMusic = MusicAPI()
+    
     @Published var musicList: [Music] = []
-    var progress: Binding<Bool>
     
-    init(progress: Binding<Bool>) {
-        self.progress = progress
-    }
+    private init() {}
     
     func getSearchResults(search: String) {
         
-        progress.wrappedValue = true
-        
         self.musicList = []
         
-        // URLSession의 싱글톤 객체
         guard var urlComponents = URLComponents(string: "https://itunes.apple.com/kr/search") else { return }
         urlComponents.query = "media=music&entity=song&term=\(search)"
-        guard let url = urlComponents.url else {
-            progress.wrappedValue = false
-            return }
+        guard let url = urlComponents.url else { return }
         
         // Networking 시작
         let task = URLSession.shared.dataTask(with: url) { [self] data, response, error in
```

Comment:
```suggestion
        let task = URLSession.shared.dataTask(with: url) { data, response, error in
```
순환참조가 일어나지 않는 것 같은데.. 맞나요? ? ?

Reply:
네 순환참조가 생기지는 않습니다!
보기 좋게 삭제해도 좋을 것 같네요!

꼼꼼하게 확인해주셔서 감사해요 쏘이~